wp-content/plugins/tapestry/controller/class.tapestry-permissions.php

```diff
@@ -19,4 +19,12 @@ static function putTapestrySettings($request) {
         }
         return current_user_can('publish_posts');
     }
+
+    static function postTapestryNode($request) {
+        $postId = $request['id'];
```

Comment:
Doesn't this index also need to be changed to `tapestryPostId`?

Reply:
Oh yes, I missed that. Will change it now.